src/test/weightedAnts.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  faker:
+    insectsfromfile:
+      weightedAnts:
+        - value: "Driver ant"
+          weight: 6.0
+        - value: "Fire ant"
+          weight: 3.0
+        - value: "Harvester ant"
+          weight: 1.0
```

Comment:
With simple example it works
however what it we have more complex structure like 
https://github.com/datafaker-net/datafaker/blob/73ae1267e053dd9cb93039b1c721fdf0d282996b/src/main/resources/en/planet.yml#L14-L71

Reply:
Since it was agreed that no changes should be introduced to `FakeValuesService.java`, I removed the `fetchWeightedList` method. Consequently, interacting with YAML files is no longer supported so I have also removed the example YAML file.